src/CMakeLists.txt

```diff
@@ -138,13 +138,13 @@ IF (NOT CLIENT_ONLY)
     robot.impl.hh
     server.cc
     tools.cc # Should be moved into the library
-    LINK_DEPENDENCIES ${LIBRARY_NAME}
+    LINK_DEPENDENCIES PUBLIC ${LIBRARY_NAME} hpp-manipulation::hpp-manipulation hpp-manipulation-urdf::hpp-manipulation-urdf
```

Comment:
If you have a link to `hpp-manipulation-urdf::hpp-manipulation-urdf`, it should include a link to `hpp-manipulation::hpp-manipulation`, shouldn't it ?

Reply:
Yes, it should and I think it does. I ended up with this incrementally, realizing that hpp-manipulation was not sufficient. I will fix it.